### PR TITLE
Added and Organized Badges

### PR DESCRIPTION
**Pull request by rishichitnis007**
I organized the badges based on topic/category/purpose (ie. CI, Programming Language, etc.). I also added common and popular programming languages, frameworks, hosting services, and other things. The tables should now be formatted to match the width of the widest row.

Hope this helps!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,35 +32,56 @@ Add Badges to your Github Projects and Profiles
...
+|------------------|--------------------|-----------------------|
+| React            | <img src="https://img.shields.io/badge/react%20-%2300D9FF.svg?&style=for-the-badge&logo=react&logoColor=white"/>                 | `<img src="https://img.shields.io/badge/react%20-%2300D9FF.svg?&style=for-the-badge&logo=react&logoColor=white"/>`              |
+| Django | <img src="https://img.shields.io/badge/django%20-%23092E20.svg?&style=for-the-badge&logo=django&logoColor=white"/> | `<img src="https://img.shields.io/badge/django%20-%23092E20.svg?&style=for-the-badge&logo=django&logoColor=white"/>` |
+| Django | <img src="https://img.shields.io/badge/django%20-%23092E20.svg?&style=for-the-badge&logo=django&logoColor=white"/> | `<img src="https://img.shields.io/badge/django%20-%23092E20.svg?&style=for-the-badge&logo=django&logoColor=white"/>` |
+| TailwindCSS | <img src="https://img.shields.io/badge/tailwind-css%20-%231572B6.svg?&style=for-the-badge&logo=tailwind-css&logoColor=white"/>                 | `<img src="https://img.shields.io/badge/tailwind-css%20-%231572B6.svg?&style=for-the-badge&logo=tailwind-css&logoColor=white"/>`                             |
```
Comment (nhelchitnis) [inline README.md:40]: You have Django duplicated. Kindly take out the extra one.

**Review by nhelchitnis — CHANGES_REQUESTED** [review]
Hello👋 @rishichitnis007 !
This is an amazing contribution. Thanks for adding some more badges, and it was very thoughtful of you to organize the badges. One more thing, please attend to the change requested so that I can gladly merge into master.

File: README.md (hunk 1 above)
Comment (rishichitnis007) [reply README.md:39]: Oh sorry, fixed it

**Review by nhelchitnis — APPROVED** [review]
Awesome, thanks!

**Pull request merged**